menstruation/handlers.py

```diff
@@ -41,11 +41,14 @@
 
 def debug_handler(func):
     @functools.wraps(func)
-    def wrapper_decorator(*args, **kwargs):
-        logging.debug(f"Entering: {func.__name__}, chat_id: {args[0].message.chat_id}")
-        value = func(*args, **kwargs)
+    def wrapper_decorator(*args):
+        logging.debug(
+            f"Entering: {func.__name__}, "
+            f"chat_id: {args[0].message.chat_id}, "
```

Comment:
note this throws terrible errors when decorating other functions than command handlers

Reply:
Hehe I knew I was sailing in dangerous waters. xD